Bug fixes and User dialog modify

ZaykaNya opened the pull request:
# Pull Request ☁️
## Overview ✨
<!-- Please give a short summary about pull request changes. -->
Fix bug that makes it imposible to login or register. Added default pages for logined and logouted user. Fix bug with chip on OrganizationPageView. Modified DialogUser: new styles, new parts, delete some old parts.

## Changes 🔧
<!-- Please describe what did you do. Use closing keyword to close issues. Write module in brackets [] -->
- __[Global]__ _Added default pages and fixed add role bug when states deleted before role is added._
- __[DialogUser]__ _Modified dialog structure and styles._

## Additional context 🌟
<!-- Any additional context -->

ZaykaNya committed:
[Global] Added default pages and fixed add role bug when states deleted before role is added.

ZaykaNya committed:
[DialogUser] Modified dialog structure and styles.

DanilAndreev commented on src/layout/MainPageLayout/MainPageLayout.tsx:
```diff
@@ -38,6 +39,15 @@ const MainPageLayout = React.forwardRef((props: MainPageLayoutProps, ref: Ref<an
+    const {isLogged} = useAuth();
+    const {changeRoute} = useChangeRoute();
+
+    useEffect(() => {
```
use react-router-dom __location__ object!

DanilAndreev commented on src/layout/MonitorLayout/MonitorLayout.tsx:
```diff
@@ -80,11 +80,14 @@ const MonitorLayout = React.forwardRef((props: MonitorLayoutProps, ref: Ref<any>
     const id = openPopper ? 'simple-popper' : undefined;
 
-    console.log(isLogged);
+    useEffect(() => {
```
use react-router-dom location object!

DanilAndreev commented on src/views/OrganizationPageView/LocalComponents/DialogUser/DialogUser.tsx:
```diff
@@ -78,18 +79,76 @@ const DialogUser = React.forwardRef((props: DialogUserProps, ref: Ref<any>) => {
         setIsRemoveRoleFromUserButtonActive(null);
     }
 
+    console.log();
```
Like a boss

ZaykaNya committed:
[MainPageLayout] (PR #125) Changed document.URL to location.pathname.
[MonitorLayout] (PR #125) Changed document.URL to location.pathname.
[DialogUser] (PR #125) Deleted console.log().
